src/routes/(pages)/log/posts/2025-03-28-logging-i-nais.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Veien videre for logg i Nais
+date: 2025-03-28T14:00:00+02:00
+author: Hans Kristian Flaatten
+tags: [nais, logging, grafana, loki]
+layout: log
+---
+
+De siste årene har det vært gjort en betydelig innsats for å forbedre observabilitet i Nais. Først frontend observabilitet med Faro, og så helhetlig sporing på tvers av applikasjoner med OpenTelemetry Tracing. Alt sammen har integret sømløst med Grafana, som er Nais sitt fortrukne observabilitetsverktøy.
+
+Det har i lengre tid vært mulig for teamene å sende applikasjonslogger til Grafana. Vi tar dette nå et skritt videre og gjør det til standard i Nais. Dette vil gi deg som utvikler en helhetlig oversikt over hva som skjer i applikasjonen din, og gjøre det enklere å feilsøke og forstå hva som skjer i systemet.
+
+## Hva er nytt?
+
+Fra og med XX april 2025 vil alle applikasjoner i Nais sende logger til Grafana som standard. Dette vil gi deg tilgang til en rekke nye funksjoner og muligheter for å analysere og visualisere loggene dine, ikke minst kunne integrere logger på en enkel måte inn i Grafana Dashboards.
+
+I en overgangsperiode vil det være mulig å sende logger til både Grafana og Elastic for å sikre en smidig overgang.
```

Comment:
Denne setningen gjelder kun Nav, er det sant for hele annonseringen? Burde vi da gjøre det tydeligere? Og er det ikke slik at noen tenants ikke ønsker å bruke vårt oppsett for logging også?